NoSQL/week1/README.md

```diff
@@ -3,21 +3,29 @@
 This week will focus on three topics: learning how to use Git Bash and Github, creating databases, and accessing databses.
 
 ### 📚 Summary
```

Comment:
Summary and Learning Objectives are the same as the main readme.md [here](https://github.com/emsesc/BitCamp/tree/nosql/NoSQL). Comment on [this PR](https://github.com/bitprj/BitCamp/pull/56) if you think they should be different/want to incorporate something else)